test/e2e/standard/run-tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+set -o pipefail
+
+serverUrl="http://localhost:8383"
+userEmail="x@example.com"
+userPassword="secret1234"
+
+log() { echo "[test/e2e/standard/run-tests] $*"; }
+
```

Comment:
a log entry here would helpful. When I executed this script for the first time, I didn't see anything on the console for few seconds, which made me wonder if anything is happening at all